packages/dev/docs/src/docs.css

```diff
@@ -89,18 +89,18 @@ article {
   }
 }
 
-article h1 {
+article h1.articleHeader  {
   margin-top: 0 !important;
```

Comment:
I left this `!important` in because it was the easiest solution and the className specificity should prevent this from impacting the components.